tests/services/data.service.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { of } from 'rxjs';
+import { DataService } from '../../src/app/core/services/data.service';
+import { GithubService } from '../../src/app/core/services/github.service';
+import { autoSpy } from '../auto-spy';
+import { csvString, dataFile, dataFileTeamStructure } from '../constants/data.constants';
+
+describe('DataService', () => {
+  it('when getDataFile is called it should return the correct data', done => {
+    // arrange
+    const { build, githubService } = setup().default();
+    const c = build();
+    githubService.fetchDataFile.and.returnValue(of({'data': csvString}));
+    // act
+    const d = c.getDataFile();

```

Comment:
Same here; I think using a more descriptive name (rather than `d`) will help us when we edit these tests in future